src/AbstractStore.php

```diff
@@ -63,9 +63,13 @@ public function fetch($url)
 
         $ttl = 0;
         if ($res->hasHeader('Cache-Control')) {
+            $cacheControl = $res->getHeader('Cache-Control');
+            if (is_array($cacheControl)) {
```

Comment:
Are you looking at Guzzle documentation? Not sure what that says, actually.

There's [PSR-7: HTTP message interfaces](http://www.php-fig.org/psr/psr-7/), which describes it.